python/plugins/processing/gui/AlgorithmDialog.py

```diff
@@ -200,7 +200,7 @@ def accept(self):
             self.setProgressText(QCoreApplication.translate('AlgorithmDialog', 'Processing algorithm…'))
 
             self.setInfo(
-                self.tr('AlgorithmDialog', '<b>Algorithm \'{0}\' starting&hellip;</b>').format(self.algorithm().displayName()), escapeHtml=False)
+                QCoreApplication.translate('AlgorithmDialog', '<b>Algorithm \'{0}\' starting&hellip;</b>').format(self.algorithm().displayName()), escapeHtml=False)
```

Comment:
I think we can make translator's lifes easier by specifying strings like this as 

```
'<b>{}</b>'.format(QCoreApplication.translate('AlgorithmDialog', 'Algorithm \'{0}\' starting…').format(self.algorithm().displayName()))
```

I know you've been fixing something else here, more wondering for this in general